internals/cli/env.go

```diff
@@ -79,6 +83,18 @@ func (a *App) unregisterEnvVar(name string) {
 	delete(a.knownEnvVars, strings.ToUpper(name))
 }
 
+// ExtraEnvVarFunc takes a function that determines additional environment variables
+// recognized by the application.
+func (a *App) ExtraEnvVarFunc(f func(key string) bool) *App {
+	isExtraEnvVar := a.isExtraEnvVar
+	if f != nil {
+		a.isExtraEnvVar = func(key string) bool {
+			return isExtraEnvVar(key) || f(key)
```

Comment:
The first part of this boolean statement always seems to return false. What's going on here?

Reply:
When calling `ExtraEnvVarFunc` several times, I'd expect a variable that's matched by any of the functions to be recognized.

It might be more readable to store the functions in an array and iterate over them, instead of combining them into one function. I'll update this.

Reply:
bc5c9c36ad0c52ee3f07cb6a4e15a60c600562c3